skimage/morphology/rolling_ball.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+import numpy as np
+
+from skimage.util import invert
+
+
+def rolling_ball(input_img, radius=50, white_background=False):
+    """Perform background subtraction using the rolling ball method.
+
+    The rolling ball filter is a segmentation method that aims to separate the
+    background from a grayscale image in case of uneven exposure. It is
+    frequently used in biomedical image processing and was first proposed by
+    Stanley R. Sternberg (1983) in the paper Biomedical Image Processing [1]_.
+
+    Parameters
+    ----------
+    image : ndarray
+        The image to be filtered.
+    radius : float
+        The radius of the ball that is used as the structuring element.
+    white_background : bool
```

Comment:
```suggestion
    white_background : bool, image
```

Reply:
What do you mean by adding "image" as allowed input to `white_background`?